pinot-integration-tests/src/test/resources/test_null_handling.schema

```diff
@@ -1,4 +1,10 @@
 {
+  "options": {
+    "nullHandling": {
+      "mode": "column",
+      "default": true
```

Comment:
- assume ***mode*** can either be `column` or `table`?
- so we plan to allow users to configure the ***default nullability*** on a table?

Reply:
Does this still apply? Should we set "enableColumnBasedNullHandling" instead?

Reply:
^^
Also, how does it work right now? Do we have test that rely on column based null handling to be enabled?